src/utils/validateBurnedAmount/validateStaticBurnedAmount.ts

```diff
@@ -1,26 +1,17 @@
 import type { Context } from '../../vms/context/model';
 import {
-  isAddDelegatorTx,
-  isAddPermissionlessDelegatorTx,
-  isAddPermissionlessValidatorTx,
-  isAddSubnetValidatorTx,
-  isAddValidatorTx,
-  isCreateChainTx,
-  isCreateSubnetTx,
   isPvmBaseTx,
   isExportTx as isPvmExportTx,
   isImportTx as isPvmImportTx,
   isRemoveSubnetValidatorTx,
   isTransferSubnetOwnershipTx,
-  isTransformSubnetTx,
```

Comment:
Should we remove all pvm tx types from the static burned amount checker? We shouldn't be hitting those codepaths anymore?